src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -44,7 +43,6 @@ public abstract class RazorPage : IRazorPage
         public RazorPage()
         {
             SectionWriters = new Dictionary<string, RenderAsyncDelegate>(StringComparer.OrdinalIgnoreCase);
```

Comment:
Change for change (and c# 6.0) sake. I did something similar when changing from `_writerScopes` to `_tagHelperScopes` but the replacement was new code. Don't think it's worthwhile for `SectionWriters`.